### PR TITLE
home-assistant: 2021.1.1 -> 2021.1.2

### PR DESCRIPTION
**Pull request by mweinelt**
###### Motivation for this change

https://github.com/home-assistant/core/releases/tag/2021.1.2

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

@GrahamcOfBorg test home-assistant

### CODE REVIEW

File: pkgs/servers/home-assistant/component-packages.nix [hunk 1]
```diff
@@ -897,7 +897,7 @@
     "vera" = ps: with ps; [ ]; # missing inputs: pyvera
     "verisure" = ps: with ps; [ jsonpath ]; # missing inputs: vsure
     "versasense" = ps: with ps; [ ]; # missing inputs: pyversasense
-    "version" = ps: with ps; [ pyhaversion ];
+    "version" = ps: with ps; [ ]; # missing inputs: pyhaversion
```
Comment (mweinelt) [inline pkgs/servers/home-assistant/component-packages.nix:900]: This is somehow due to 0b371941a1d8044fa09919a30daaeb00c55a8f16. @fabaff

Comment (fabaff) [reply pkgs/servers/home-assistant/component-packages.nix:900]: Yes, it looks that way.

Comment (mweinelt) [reply pkgs/servers/home-assistant/component-packages.nix:900]: You disabled the package for python >= 3.8.

**Comment by mweinelt** [comment]
Test case is working on x86_64-linux, deployed to aarch64-linux. WFM.

**Pull request merged**